File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -703,6 +715,562 @@ End Using
 
 {% endtabs %}
 
+## Table of Figures 
+
+The following code example illustrates how to set the name of the sequence identifier employed in the creation of a table of figures using TableOfFiguresLabel API. 
```

Comment:
The following code example shows how to add a table of figures in Word document using TableOfFiguresLabel API.